src/cli/cluster/worker.js

```diff
@@ -82,8 +85,15 @@ module.exports = class Worker extends EventEmitter {
   }
 
   onMessage(msg) {
-    if (!_.isArray(msg) || msg[0] !== 'WORKER_BROADCAST') return;
-    this.emit('broadcast', msg[1]);
+    switch (_.isArray(msg) && msg[0]) {
+      case 'WORKER_BROADCAST':
```

Comment:
I gave this a shot and it really only complicated things. If you can make a good case I might be convinced.